source/source_integration_test.go

```diff
@@ -30,6 +30,15 @@ import (
 	"github.com/matryer/is"
 )
 
+const createTableQueryFmt = `
+		CREATE TABLE %s (
```

Comment:
could you please format to 
```
CREATE TABLE % s (
  id NUMBER GENERATED by default on null as IDENTITY, 
  name VARCHAR2(30) NOT NULL, 
  age NUMBER, 
  is_active NUMBER(1, 0), 
  PRIMARY KEY (id)
)

```
I use https://codebeautify.org/sqlformatter for this case